service/service.go

```diff
@@ -272,6 +274,21 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var remotewriteController *remotewrite.Controller
+	{
+		c := remotewrite.ControllerConfig{
+			K8sClient:        k8sClient,
+			Logger:           config.Logger,
+			PrometheusClient: prometheusClient,
+		}
+
+		remotewriteController, err = remotewrite.NewController(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+	}
```

Comment:
I would rather leave this out for now, and add it at last when we have resources and when everything works. So we keep PMO available for any upcoming and unforeseen fixes needed.

Also we this you need to boot the controller, basically adding a line further down in this file in the `Boot` method.

Reply:
Agree, `remotewriteController` would wired in a separate PR.